src/freenet/l10n/freenet.l10n.fi.properties

```diff
@@ -908,7 +898,6 @@ PproxyToadlet.pluginNotFoundReloadTitle=Liitännäistä ei löydetty (uudelleenl
 PproxyToadlet.pluginNotFoundUpdatingTitle=Liitännäinen ei löytynyt (päivitetään)
 PproxyToadlet.pluginNotFoundUpdating=Liitännäinen, jota yritit päivittää, ei latautunut!
 PproxyToadlet.pluginSourceChoice=Voit noutaa lisäosan Freenetistä tai keskuspalvelimiltamme:
-PproxyToadlet.pluginSourceFreenet=Hae Freenetistä. Tämä ei ole jäljitettävissä ja turvallista, mutta voi olla hidasta.
```

Comment:
This is strange. I’m not sure why this got deleted from the translations. I’ll have to go through the translation changes again to see whether they are all OK.

Reply:
I'm not sure how familiar you're with the way our l10n works so here is the most simple explanation:
Whenever an English source string changes we remove all existing translations of it so there always either is an up-to-date translation or none. Transifex does implement that IIRC.
So perhaps the English source string changed?

By the way, why are you still able to update translations of a master-based build anyway? Shouldn't Transifex already be updated to contain the English source strings for next, not master, and hence updating translations should be invalid until we deploy next?

Reply:
I’m able to use that, because next and master differ by very little. I did all possible backports to avoid blocking our moving forward by stuff which blocks next.

It’s possible that the English source changed, yes. Thank you for the explanation!